ecs-agent/netlib/platform/api.go

```diff
@@ -49,10 +51,21 @@ type API interface {
 	// GetNetNSPath returns the path of a network namespace.
 	GetNetNSPath(netNSName string) string
 
-	ConfigureInterface(ctx context.Context, netNSPath string, iface *networkinterface.NetworkInterface) error
+	// ConfigureInterface configures an interface inside a network namespace
+	// for it to be able to serve traffic.
+	ConfigureInterface(
+		ctx context.Context,
```

Comment:
nit: doesn't look gofmt ed. Should we align the spaces here?

Reply:
Thanks for pointing it out. Will fix it.

Reply:
I checked my IDE and looks like I am keeping the indentation as 4 spaces per tab. While viewing other pages on github, we see the same pattern where tab length seems to be 8 spaces instead of 4. So I think we do not have to change anything here. Besides lint checks pass, so I think we are good.